### DISCUSSION TITLE
Cult 4.0 Megathread

### DISCUSSION THREAD

**Discussion by jknpj** [vgstation-coders/vgstation13#28636] [General]
From @DeityLink 's original issue

"Alright good, got your attention. I stayed up till late last night to finish up this google doc summing up some of the ideas we discussed a bit over a week ago following over a year and a half of feedback on cult 3.0. This includes my plan to "fix" cult, and the reasoning behind each step. Please give it a read, and come discuss it in this issue so I can make adjustments to the document as needed and start coding ASAP."

Doc:
# [Cult 2020 Design Doc](https://docs.google.com/document/d/1e49TxEMFdzAwpvIy6alBgYrytWfAo5F-O5A62PW2KvY/edit?usp=sharing)

## [Cult 2020 - Rituals List](https://docs.google.com/document/d/1rlOxtkmFLOlVJVRo1jeVumdv8Tc_VcdAIqkk-0BwvOQ/edit#)

Related Issues:
#26941

Related PRs:
#28613 #28084 #27950 #28868 #28806

**Comment by PavelCIA** [comment]
sneed

**Reply by DeityLink** [reply]
Hey buddy I think you got the wrong door, the cringe club is two blocks down.

**Reply by gurfan** [reply]
wtf cringe chuck

**Comment by DeityLink** [comment]
I'll say that we probably don't need this thread anymore, most of the planned content was finally added with #36361
